ocs_ci/utility/rosa.py

```diff
@@ -893,6 +894,23 @@ def destroy_appliance_mode_cluster(cluster):
     return True
 
 
+def destroy_rosa_cluster(cluster, best_effort=True):
+    """
+    Delete rosa cluster
+
+    Parameters:
+        cluster (str): name of the cluster
+        best_effort (bool): If True (true), ignore errors and continue with the deletion of the cluster
+    """
+    external_id = get_cluster_details(cluster)["id"]
+    cmd = f"ocm delete cluster {external_id} -p best_effort={str(best_effort).lower()}"
+    proc = exec_cmd(cmd, timeout=1200)
+    if proc.returncode != 0:
+        raise CommandFailed(f"Failed to delete cluster: {proc.stderr.decode().strip()}")
+    logger.info(f"{proc.stdout.decode().strip()}")
```

Comment:
It might be worth to put this log message before raising the exception, to be sure the stdout is logged even when the command fails.